x-pack/plugins/security_solution/common/api/detection_engine/prebuilt_rules/review_rule_upgrade/review_rule_upgrade_route.ts

```diff
@@ -21,6 +21,12 @@ export interface RuleUpgradeStatsForReview {
   /** Number of installed prebuilt rules available for upgrade (stock + customized) */
   num_rules_to_upgrade_total: number;
 
+  /** Number of installed prebuilt rules with upgrade conflicts (SOLVABLE or NON_SOLVALBE) */
+  num_rules_with_conflicts: number;
+
+  /** Number of installed prebuilt rules with NON_SOLVABLE upgrade conflicts */
+  num_rules_with_non_solvable_conflicts: number;
```

Comment:
Why do we need this information in the `upgrade/_review` response?

Reply:
The use case I have in my mind for these two props is the [Add ability to bulk update all rules that don't have conflicts #180589](https://github.com/elastic/kibana/issues/180589).

We'll want a bulk update button that opens a modal with text that looks like:

`${num_rules_to_upgrade_total} Elastic rules have updates. ${num_rules_to_upgrade_total - num_rules_with_non_solvable_conflicts} will be updated automatically, but ${num_rules_with_non_solvable_conflicts} have conflicts that will need manual resolution`.